taf/keys.py

```diff
@@ -236,11 +237,14 @@ def load_signing_keys(
     # if the keystore file is not found, ask the user if they want to sign
     # using yubikey and to insert it if that is the case
 
-    keystore_path = None
-    if keystore is not None:
-        keystore_path = Path(keystore).expanduser().resolve()
-    else:
-        taf_logger.info("Keystore location not provided")
+    if keystore is None:
+        keystore_path = find_keystore(Path.cwd())
```

Comment:
`find_taf_directory` expects `auth_repo_path`, so this couldn't find the keystore when I was in the auth repo's parent directory. Everything worked when I was in `D:\oll\library\oll-test-repos\cityofsanmateo\law`, but not when I was in ``D:\oll\library\oll-test-repos\cityofsanmateo` (the repository was found, keystore was not)